packages/keyring-api/docs/sol-methods.md

```diff
@@ -0,0 +1,401 @@
+# SOL Methods
+
+Here we document the Solana methods that an account Snap may implement to
+support requests originated from dapps.
+
+## [signAndSendTransaction](https://github.com/anza-xyz/wallet-standard/blob/master/packages/core/features/src/signAndSendTransaction.ts#L10)
+
+Signs and sends a transaction to the Solana blockchain.
+
+### [Parameters](https://github.com/anza-xyz/wallet-standard/blob/master/packages/core/features/src/signAndSendTransaction.ts#L42)
+
+- **Transaction intent (required)**
+  - Type: `object`
+  - Properties:
+    - `origin` (required)
```

Comment:
Team, if the origin if MetaMask, should we send `metamask.io`? or should we send a `metamask` string

Reply:
The MetaMask origin should be `metamask`, it's the origin we use elsewhere.

But it shouldn't be part of the signing request itself because it needs to be set by the **client** in a chain-agnostic way. I think it should be added to the [`KeyringRequest`](https://github.com/MetaMask/accounts/blob/main/packages/keyring-api/src/api/request.ts) type.

Reply:
so next to the nested `request`?

Reply:
Removed, then to be added on the `KeyringRequest`